tests/rhoai/scripts/gpu-setup.sh

```
@@ -0,0 +1,208 @@
+# Apply ClusterPolicy
+echo ""
+echo "--> Applying ClusterPolicy..."
+oc apply -f "$BASE_DIR/manifests/gpu/cluster-policy.yaml"
+
+# Wait for GPU operator pods to be created and healthy
+echo ""
+echo "--> Waiting for GPU operator pods to be healthy..."
+echo "    This may take up to 10 minutes while images are pulled and pods start..."
+timeout=1200
+elapsed=0
+until oc get pods -n nvidia-gpu-operator --no-headers 2>/dev/null | awk '{if ($3 != "Running" && $3 != "Completed") exit 1}' && [ $(oc get pods -n nvidia-gpu-operator --no-headers 2>/dev/null | wc -l) -gt 5 ]; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)